### ISSUE TITLE
Crash when image field is nil

### ISSUE THREAD

**Issue by afpava** [zooppa/administrate-field-carrierwave#12]
Hello!
I have a problem, in my database is a string field `avatar`  where `carrierwave`  stores image path.
But when field is `nil`  `administrate` crashes when I click to `show` page of a User, avatar belongs to with the error: 

```
Showing /home/--/.rvm/gems/ruby-2.5.1/gems/administrate-field-carrierwave-0.4.0/app/views/fields/carrierwave/_file.html.erb where line #1 raised:
undefined method `split' for nil:NilClass

<%= link_to (file.filename || file.path).split('/').last, file.url, target: '_blank' %>

```
User was generated from console, but I think that's not the problem.. 
 `Carrierwave` is configured to store files locally.
Version of  `administrate-field-carrierwave` is  0.4.0
Settings: 
```
ATTRIBUTE_TYPES = {
  bar: Field::Carrierwave.with_options(
    image: :standard,
    multiple: true,
    image_on_index: true,
    remove: false,
    remote_url: false
  )
}.freeze
```
Changing ` _file.html.erb` to 
```

<%= link_to (file.filename || file.path).split('/').last, file.url, target: '_blank' if file.filename || file.path%>

```
hopefully resolves the issue.

A little update on the issue: 
`multiple: true`, - cause the issue, if set `multiple: false` - everything works as it should

```
<%= link_to image_tag(file.url, class: 'img-circle img-responsive',), file.url, target: '_blank' if file.url %>
```
This way I add image preview along with the link, although style is another question (what styles available in administrate to show thumbnail, but that's not the issue  ).

I thought I missread doccumentation, tried to add method to the `administrate` `ApplicationControler` - but anyway it crashes with `multiple: true`.
 
Thank you!

**Comment by michelegera** [comment]
Hey @afpava, thanks for your report. Can you try using the `fix-file-preview` branch I just pushed and see if it fixes your issues?

You might want to change the Gemfile of the project where you are experiencing this issue with:

```
gem 'administrate-field-carrierwave', github: 'zooppa/administrate-field-carrierwave', branch: 'fix-file-preview'
```

then run `bundle`, restart your server and test.

Let me know how it goes, if it fixes the error I will merge it and release a new version of the gem.

**Comment by afpava** [comment]
Hello @michelegera, thank you for the quick fix :).
Everything works as it should - no errors this time.
I've been reading the code, and ran into the 18-th line in `_index.html.erb` 
and found this partial render.
` <%= render 'fields/carrierwave/preview', file: field.file, field: field %>`, but I don't see that partial in the views folder.

Thank you, you've done a marvelous job!

**Comment by michelegera** [comment]
Hey @afpava, thanks for the feedback! I’ve just incorporated the change in the master branch and released [v0.4.1](https://rubygems.org/gems/administrate-field-carrierwave/versions/0.4.1) of the gem.

Good catch on that partial render too! It was a leftover from a previous feature that was removed in d79596bd, so I cleaned that up as well.
